IPs/RSKIP417.md

```diff
@@ -0,0 +1,46 @@
+---
+rskip: 417
+title: Improve empty Bridge response when called from a smart contract
+created: 28-FEB-24
+author: MI
+purpose: Usa
+layer: Core 
+complexity: 1
+status: Draft
+description: 
+---
+
+|RSKIP          |417           |
+| :------------ |:-------------|
+|**Title**      |Improve empty Bridge response when called from a smart contract |
```

Comment:
Suggesting another name: Avoid transactions to be reverted when Bridge method calls from smart contracts return an empty response. Is it too long?